packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -118,7 +166,53 @@ function reframeAsObjectInPlace(
   }
 }
 
-function fixObjectPropertiesInline(
+function isConforming(
+  client: MinimalClient,
+  def:
+    | InterfaceDefinition<any, any>
+    | ObjectTypeDefinition<any, unknown>,
+  obj: OntologyObjectV2,
+  propsToCheck: readonly string[],
+) {
+  for (const propName of propsToCheck) {
+    if (process.env.NODE_ENV !== "production") {
```

Comment:
I'm probably missing something here, but wouldn't you only want to log this when you find something where its non-nullable and the prop is null? So like inside the if on line 190?